pkg/starboard/plugin.go

```diff
@@ -44,6 +46,9 @@ type PluginContext interface {
 	// GetServiceAccountName return the name of the K8s Service Account used to run workloads
 	// created by Starboard.
 	GetServiceAccountName() string
+	// GetOperatorConfig return the starboard operator config instance, with which starboard operator
+	// is running.
+	GetOperatorConfig() etc.Config
```

Comment:
See my comment related to PluginFactory and etc.Config. I think instead we could pass (generic) Starboard config to the PluginContext.